forem/forem
# Fixed twitter timeline display in comments preview

## What type of PR is this? (check all applicable)


## Description
The tweet timeline/tweet itself does not show up when we add a tweet_timeline tag in a comment under an article and hit Preview. The only thing that shows up is the CTA button view on twitter. This PR fixes the issue.

## Related Tickets & Documents
Closes #12039 

## QA Instructions, Screenshots, Recordings

- Go to any article
- Scroll down to comment section
- Copy and paste this code:

{% twitter_timeline https://twitter.com/NYTNow/timelines/576828964162965504 %}

- Click Preview button
- Observe that the tweet timeline shows up along with the CTA view on twitter button, as opposed to just the CTA button showing up in master

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?

      have not been included_

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
